src/controllers/users.js

```diff
@@ -115,6 +118,30 @@ export default class usersController {
       });
     }
   }
+  static async logout(req, res) {
+    try {
+      req.logout();
+      var log = localStorage.getItem("token");
+      if (!log)
+      {
+        return res.status(401).json({
+          status: 401,
+          error: 'you are not logged in',
```

Comment:
I think you should create a middleware to check user if is loggedIn or registered  in order to acess this route.

Reply:
thx @EugeneMunya, I don't know if I got well your question, but I think it is not necessary because if you have well followed the flow of the code it check-in only in the localStorage variable if there is something in it, and if not it will give you that message   ` error: 'you are not logged in'`.

Reply:
thank you @EugeneMunya  what you were telling me I saw that it is a good way to implement that and I have done it know you can again pull the changes. thank you